docs/book/src/topics/capibmadm/powervs/port.md

```diff
@@ -0,0 +1,21 @@
+## Power VS Network Port Commands
+
+### 1. capibmadm powervs port create
+
+#### Usage:
+Create Power VS network port.
+
+#### Environmental Variable:
+IBMCLOUD_API_KEY: IBM Cloud api key.
+
+#### Arguments:
```

Comment:
Missing required arguments `--network-id`. `--description` can also be added here.